hack/update-docs-and-licenses.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright (c) Codesphere Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+IFS=$'\n\t'
+
+here=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+root=$(cd "$here/.." && pwd)
+
+echo "Working directory: $root"
+
+cd "$root"
```

Comment:
why do we need that? I assume the workflow is running in the repo root directory, otherwise `./hack/update-docs-and-licenses.sh` would fail